tests/_cli.py

```diff
@@ -0,0 +1,13 @@
+import subprocess
+
+
+def run(node, command):
+    p = subprocess.Popen(
+        ['docker', 'exec', 'kubevirt-' + node, 'ssh.sh'] + command,
+        stdin=subprocess.PIPE,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+    )
+    stdout, stderr = p.communicate()
+
+    return p.returncode, stdout.split('\n')[:-1], stderr.split('\n')[2:-1]
```

Comment:
It will be better to raise and add all the info needed there.
When all is working as expected, you do not need to handle any asserts on the success code.